src/components/general/Button/styles.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components'
+
+export const StandardButton = styled.button`
```

Comment:
why do you need standard button and then extend it to invercolourbutton, cant it just be one button? 
you could argue its more readable, up to you